org.mixedrealitytoolkit.input/Tracking/TrackedPoseDriverExtensions.cs

```diff
@@ -84,14 +89,27 @@ internal static InputTrackingState GetInputTrackingStateNoCache(this TrackedPose
                 return InputTrackingState.None;
             }
 
+            InputTrackingState result = InputTrackingState.None;
             if (trackingStateAction.HasAnyControls())
             {
-                return (InputTrackingState)trackingStateAction.ReadValue<int>();
+                result = (InputTrackingState)trackingStateAction.ReadValue<int>();
             }
             else
             {
-                return InputTrackingState.None;
+                result = InputTrackingState.None;
             }
```

Comment:
nit: this `else` could be removed now that `result` is already initialized to `InputTrackingState.None`